libsolidity/analysis/TypeChecker.cpp

```diff
@@ -322,6 +322,16 @@ bool TypeChecker::visit(StructDefinition const& _struct)
 	return false;
 }
 
+bool TypeChecker::visit(ModifierDefinition const& _modifier)
```

Comment:
Can you use `endVisit` instead?

Reply:
In general, `endVisit` is safer, because newly added nodes in the ast will be automatically visited.